Cargo.toml

```diff
@@ -6,7 +6,8 @@ members = [
     "components/zcash_encoding",
     "components/zcash_protocol",
     "components/zip321",
-#    "zcash_client_backend",
+    # zcash_client_backend and zcash_client_sqlite are not compatible with OrchardZSA
```

Comment:
It is also commented in CI workflows. Specifically - workflows/ci.yml:L196-198

Shall we ucomment there too?

Reply:
That makes sense, thanks @alexeykoren, I just fixed it and commited the change.